templates/common/infra/bicep/core/host/container-app.bicep

```diff
@@ -71,8 +107,15 @@ resource containerRegistry 'Microsoft.ContainerRegistry/registries@2022-02-01-pr
   name: containerRegistryName
 }
 
+// user assigned managed identity to use throughout
+resource managedIdentityRes 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' existing = {
```

Comment:
Do you need Res in name

Reply:
I needed to differentiate between `managedIdentityRes`(Resource) and the `managedIdentity` deprecated flag. If `managedIdentity` was not a flag would have used that name instead. Do you have any recommendations for the name?

Reply:
using `userManagedIdentity` for now.